app/models/concerns/hydranorth/generic_file/doi.rb

```diff
@@ -0,0 +1,15 @@
+module Hydranorth
+  module GenericFile
+    module DOI 
+      extend ActiveSupport::Concern
+      include Linkable
+      included do
```

Comment:
I don't think we need to define an included block if it's empty?, for the same reason, it's probably unnecessary to extend ActiveSupport::Concern if we're not taking advantage of it.